src/api-doc/api-doc.yaml

```diff
@@ -3,9 +3,9 @@ info:
   title: Entity-Management-Service
   version: 1.0.0
   description: >-
-    * This Service enables the creation and management of various entities and entityType .It
-    provides functionalities for entities, ensuring seamless integration and
-    maintenance of entity-related data across the platform.
+    * This Service enables the creation and management of various entities and
+    entityType .It provides functionalities for entities, ensuring seamless
```

Comment:
@Prajwal17Tunerlabs should we include same tabular structure you added for project service? please check once